src/Http/Http.Results/src/ResultsCache.StatusCodes.tt

```diff
@@ -0,0 +1,108 @@
+<#@ template debug="true" hostspecific="true" language="C#" #>
```

Comment:
I imagine this limits you to having to use VS to regen the file. Have you considered doing something different (e.g. https://github.com/dotnet/aspnetcore/tree/main/src/Servers/Kestrel/tools/CodeGenerator) which works everywhere? Alternatively now that the generated code is available, use that to drive further work and not check this in?

Reply:
If I am not wrong it is already supported in `Rider` and probably should have some `VS Code extension` available but I could check the alternative that you mentioned.

Reply:
I think it's fine to use t4.

Reply:
Also, a quick search let me find the `Mono.TextTemplating` project https://github.com/mono/t4. That is a community `dotnet` tool that allows the transformation of the T4 templates.

Reply:
Yes, @mhutch owns this